.github/workflows/alpha-release.yml

```diff
@@ -4,6 +4,11 @@ on:
   # Allows running workflow manually from the GitHub Actions tab
   workflow_dispatch:
 
+# cancel workflow when a newer version of the workflow is triggered on the same github ref
```

Comment:
This should only run once a day? as well should we merge changes into main wouldn't that cancel the action as well? or does this only get cancelled if someone manually kicks off alpha?

Reply:
After reviewing your PR, I stopped myself after spending 10 minutes looking at ways to generate the github actions YAML files from starlark / JS and checking the generated files in.

I really want to dedupe some of our GH actions files, but I'm unsure if using non-standard tooling is worth the deduplication (there's learning cost for folks on the team).

Perhaps we should explore composition by writing (local) actions for common stuff: https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action